iOS/Flows/Room/RoomCollectionViewLayout.swift

```diff
@@ -42,9 +42,29 @@ class RoomCollectionViewLayout: UICollectionViewCompositionalLayout {
                 let section = NSCollectionLayoutSection(group: group)
                 section.contentInsets = NSDirectionalEdgeInsets(top: sectionInset,
                                                                 leading: 0,
-                                                                bottom: 20,
+                                                                bottom: 0,
                                                                 trailing: 0)
+                section.visibleItemsInvalidationHandler = { (items, offset, environment) in
```

Comment:
Nice effect, and gives me an index update anytime distance from center is less than 5